src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -84,7 +100,7 @@ public static Type parseStatus(String status) throws ParseException {
         requireNonNull(status);
         String trimmedStatus = status.trim();
         if (!Type.isValidType(trimmedStatus)) {
-            throw new ParseException(Type.MESSAGE_TYPE_CONSTRAINTS);
+            throw new ParseException(Post.MESSAGE_POST_CONSTRAINTS);
```

Comment:
This line not supposed to be changed to Post.MESSAGE_POST_CONSTRAINTS.
revert to Type.MESSAGE_TYPE_CONSTRAINTS)